### ISSUE TITLE
Rtl ScrollY Problem

### ISSUE THREAD

**Issue by MmKargar** [thedevdojo/voyager#3655]
- Laravel: v5.6
 - Voyager: v1.1
 - PHP: 7.2
 - Database: MySQl
hey guys . 
in rtl version voyager have css issue.
can anyone fix this problem?

![image](https://user-images.githubusercontent.com/11253027/47253035-ad091980-d45a-11e8-94bd-10fc3359182c.png)

in rtl version when my menus are too many and needs scroll , scrolling not showing very well and show the scroll bar on my menues.
i tried to fix it but i couldn't.
if anyone can , please help.

**Comment by emptynick** [comment]
I can see what you describe, sometimes its working, sometimes its not.
Looks like the position is calculated somewhere in Javascript and it misses the changes to the menu-width sometimes.

**Comment by MmKargar** [comment]
exactly this is javascript calculation , sometimes its work fine in chrome and IE but always showing it in mozilla.
and i try so much but it didn't solved. :neutral_face:

**Comment by mohamed4shim** [comment]
i have the same problem when page first load, and it's hidden after clicking any link,any one can help me?

**Comment by MmKargar** [comment]
the problem didn't solved and i just remove perfect scrollbar from voyager and use [simplebar](https://github.com/Grsmto/simplebar) its not rtl but its not so bad as perfect scrollbar rtl.
![image](https://user-images.githubusercontent.com/11253027/47964636-12abf700-e052-11e8-929e-fa0030a34e88.png)

i try so many rtl javascript scrollbars libraries but all of them have conflict.
perfect scrollbar and other scrollbars has no problem , the problem is conflict with voyager.

**Comment by deva7mad** [comment]
hay @mitismirza how are you.
 can you explain how you fixed it ?

**Comment by MmKargar** [comment]
@deva7mad  you need to remove perfectScroll from voyager . for that you have to clone the voyager and remove 
```
   window.perfectScrollbar = require('perfect-scrollbar/jquery')($);

    $('.side-menu').perfectScrollbar();
```
from voyager/resources/assets/js/ app.js 

 and remove 

```
@import "node_modules/perfect-scrollbar/dist/css/perfect-scrollbar";

/********** Add z-index to perfect-scrollbar **********/

.ps.ps--active-x>.ps__scrollbar-x-rail, .ps.ps--active-y>.ps__scrollbar-y-rail {
    z-index: 999;
}

/********** END perfect-scrollbar style **********/
```

from voyager/resources/assets/sass/app.scss   and make production files again with laravel Mixer. 

or you can replace this files to public/tcg/voyager/assets/js and public\vendor\tcg\voyager\assets\css in your project that i remove the perfect scroll.
[app.css](https://www.dropbox.com/s/latunpjimh12vov/app.css?dl=0)
[app.js](https://www.dropbox.com/s/8w70uuecwmf9q4d/app.js?dl=0)

after than you can use [simplebar](https://github.com/Grsmto/simplebar) or other beauty scrollbars.
and add them to sidebar.blade.php in side-menu class .

**Comment by emptynick** [comment]
Looks like this will be fixed in https://github.com/the-control-group/voyager/pull/3832 because of an updated version of Perfect-Scroll.

**Comment by emptynick** [comment]
Closing as #3832 was merged (into 1.x, to be released in a future 1.2 release).
Let us know if it doesn't work (remember - in a future 1.2 release)
